openmetadata-ui/src/main/resources/ui/src/constants/profiler.constant.ts

```diff
@@ -63,6 +63,7 @@ export const PROFILER_FILTER_RANGE = {
   last14days: { days: 14, title: 'Last 14 days' },
   last30days: { days: 30, title: 'Last 30 days' },
   last60days: { days: 60, title: 'Last 60 days' },
+  last365days: { days: 365, title: 'Last 365 days' },
```

Comment:
let's avoid adding 365 day filter here, it will impact the date range for data quality and profiler, we only need filters till 60day for data quality and profiler